README.md

```diff
@@ -1,2 +1,14 @@
 # CVScanAI
 AI Driven CV Validator
+
+## Installation
+
+1. ```pip install -r cv_scan_ai/requirements.txt```
+2. ```export OPENAI_API_KEY="sk-...."```
+
+## Testing
+
+   - Command: `python cv_scan_ai/tests/test_models.py `
+   - The goal: create a data model with corrections
+   - As input will be taken a file `data_examples/parsed_cv_example.json`
+   - As result you will see the data model
```

Comment:
Any idea why am I getting this response:
<img width="1790" alt="Screenshot 2023-08-10 at 09 34 37" src="https://github.com/COXIT-CO/CVScanAI/assets/6651168/13ed15c7-fb18-4d4f-9d8b-62d3bf4e8498">

Reply:
Why is that limitation appearing again? I can make the request from the postman as many times as I need it. Did you try to do it from postnman? What endpoint do you use for that request?